src/async-components/views/dialogs/eventindex/ManageEventIndexDialog.js

```diff
@@ -20,16 +20,14 @@ import PropTypes from 'prop-types';
 import { _t } from '../../../../languageHandler';
 
 import Modal from '../../../../Modal';
-import {formatBytes} from "../../../../utils/FormattingUtils";
+import {formatBytes, formatCount} from "../../../../utils/FormattingUtils";
```

Comment:
Hmm, so `formatCount` is designed for cramming into tiny spaces like a badge icon. For this PR, we have more space to show these numbers, so let's add a new formatting util to show the full number. I think it will give a better sense of progress that way, as you can still see numbers change regularly once you pass 1k, etc.